buddyboss/buddyboss-platform
# Privacy issue - the user is still searchable with the fields set to "Only Me"

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #502.

### How to test the changes in this Pull Request:
Steps to check the behavior in Demo:
1. Using John's account, go to Profile > Edit Profile
2. For example, set "Phone" field's visibility to "Only Me".
3. Now, using another account, try to search the phone number of John and he will be included on the search result

### Proof Screenshots or Video
https://screencast-o-matic.com/watch/cYVr1Bvsl0

### Other information:
